package.json

```diff
@@ -31,41 +31,48 @@
     },
     "homepage": "https://dailylore.com",
     "dependencies": {
+        "@babel/core": "7.5.5",
+        "@babel/preset-env": "7.5.5",
         "autotrack": "1.1.0",
-        "axios": "0.18.0",
-        "babel-core": "6.26.3",
-        "babel-loader": "7.1.5",
-        "babel-preset-env": "1.7.0",
-        "css-loader": "1.0.0",
-        "debug": "3.1.0",
+        "axios": "0.19.0",
+        "babel-loader": "8.0.6",
+        "css-loader": "3.2.0",
+        "debug": "4.1.1",
         "lodash.shuffle": "4.2.0",
         "materialize-css": "0.100.1",
-        "node-sass": "4.9.2",
-        "sass-loader": "7.1.0",
+        "node-sass": "4.12.0",
+        "sass-loader": "7.3.1",
         "storage-available": "1.1.0",
-        "style-loader": "0.22.1",
-        "webpack": "4.16.5"
+        "style-loader": "1.0.0",
+        "webpack": "4.39.2"
     },
     "devDependencies": {
-        "babel-cli": "6.26.0",
+        "@babel/node": "7.5.5",
         "clean-webpack-plugin": "0.1.19",
-        "dotenv": "6.0.0",
-        "eslint": "4.5.0",
-        "eslint-config-prettier": "2.9.0",
-        "jest": "23.4.2",
-        "jest-puppeteer": "3.2.1",
-        "mini-css-extract-plugin": "0.4.1",
-        "node-static": "0.7.10",
-        "prettier": "1.14.1",
-        "puppeteer": "1.6.2",
-        "sass-lint": "1.10.2",
-        "wait-for-localhost-cli": "1.0.0",
-        "webpack-cli": "3.1.0",
-        "webpack-dev-server": "3.1.5"
+        "dotenv": "8.1.0",
+        "eslint": "6.2.2",
+        "eslint-config-prettier": "6.1.0",
+        "jest": "24.9.0",
+        "jest-puppeteer": "4.3.0",
+        "mini-css-extract-plugin": "0.8.0",
+        "node-static": "0.7.11",
+        "prettier": "1.18.2",
+        "puppeteer": "1.19.0",
+        "sass-lint": "1.13.1",
+        "wait-for-localhost-cli": "1.1.0",
+        "webpack-cli": "3.3.7",
+        "webpack-dev-server": "3.8.0"
     },
     "babel": {
         "presets": [
-            "env"
+            [
+                "@babel/preset-env",
+                {
+                    "targets": {
```

Comment:
Necessary to fix "ReferenceError: regeneratorRuntime is not defined" when running Jest.

See https://github.com/facebook/jest/issues/3126#issuecomment-483320742